Config updated: Working per server/library scheduling with ofelia

SebbeJohansson opened the pull request:
- Removes "schedule" profile in multi server setup since it doesnt work.
- Updates image versions so that it is cohesive through the whole documentation (latest on all).
- Updates config volume path so that it is the same in multi server config as it is in the single server config.

After discussions and testing for #2139.

simonc56 commented:
We have no prove that "schedule" profile doesn not work, I suggest you remove this line in first post.
I agree to remove the `profile` stuff which is not needed and misleading users unable to read the doc.

SebbeJohansson commented:
@simonc56 The proof would be in that it doesn't work anymore on a clean system. Feel free to test it yourself.

glensc commented on README.md:
```diff
@@ -452,12 +452,11 @@ something similar to this in your `docker-compose.yml`:
 services:
   plextraktsync:
-    image: ghcr.io/taxel/plextraktsync
+    image: ghcr.io/taxel/plextraktsync:latest
```
remove this chunk

glensc commented:
Ideally, I'd see one change per pull request. lower chances that unrelated changes don't block whole pr merge. you made three changes, one of them is rejected.

SebbeJohansson commented:
@glensc Just to be clear, you are against adding ":latest" to the places where it is missing? Should we remove it from all places? I can revert the chunks here and make a new pr for how ever you want it.

SebbeJohansson commented:
@glensc okay so the documentation should have different images for different parts. Im not sure I'm a fan of that, but I'm not gonna fight it.

I have reverted the chunks you specified and added versions to the ofelia image.

glensc commented:
@SebbeJohansson, as I said, ofelia should be pinned to some major version. so if they release something breaking, the docs here would still be acurate. As for plextraktsync, it's current repository, it's latest will be supported, rather than any other major.
